libcst/helpers/expression.py

```diff
@@ -27,3 +27,23 @@ def get_full_name_for_node(node: Union[str, cst.CSTNode]) -> Optional[str]:
     elif isinstance(node, (cst.FunctionDef, cst.ClassDef)):
         return get_full_name_for_node(node.name)
     return None
+
+
+def get_full_name_for_node_or_raise(node: Union[str, cst.CSTNode]) -> str:
```

Comment:
This should probably just call `get_full_name_for_node` and then raise if it gets `None` back. That way fixes to the original function (such as my PR up now) won't get lost.